### ISSUE TITLE
Issue with finding compatible Geckodriver 0.26 for Raspberry Pi 4

### ISSUE THREAD

**Issue by 42648594** [bubonic/TGC.bundle#15]
I've seen various comments on forums stating that the version of Geckodriver 0.26 available at:

(https://github.com/mozilla/geckodriver/releases/tag/v0.26.0)

is only for x86 machines. I've done the other steps, and for geckodriver I've downloaded and moved the binary to /usr/local/bin, but it does not appear to be executable nor does chmod +x seem to make it executable. Is there any guidance on this?

**Comment by bubonic** [comment]
Yeah, it will not execute if the binary is an x86 on an ARM device. I have  not designed this agent wrt Raspberry Pi ARM in mind.

I'm sure it is doable, but you would have to find the correct GeckoDriver/Firefox/Selenium stack for the agent with Firefox and GeckoDriver binaries for ARM. Seeing as PLEX runs on a outdated python version this may prove to be very time consuming and difficult. 

I have no plans to work on a Pi release. Feel free to give it a shot if you so desire.
